build.sh

```diff
@@ -440,14 +440,25 @@ __CleanBuild=0
 __VerboseBuild=0
 __SignTypeReal=""
 __CrossBuild=0
-__ClangMajorVersion=3
-__ClangMinorVersion=5
 __MSBuildPath=$__ProjectRoot/Tools/MSBuild.exe
 __NuGetPath="$__PackagesDir/NuGet.exe"
 __DistroName=""
 __cmakeargs=""
 __OfficialBuildIdArg=
 __SkipGenerateVersion=0
+__ClangMajorVersion=3
+__ClangMinorVersion=5
+while :; do
+    if [ $# -le 0 ]; then
+        break
+    fi
+    lowerI="$(echo $1 | awk '{print tolower($0)}')"
+    if [ $lowerI  == cross ]; then
```

Comment:
Why don't you just add setting the version to the "cross" case below instead of looping over all the command line args one more time?